### PR TITLE
Updated info.json and mmd.json

### PR DESCRIPTION
**Pull request by cs-rohit-khune**
Updated info.json
- Added updated git link
- Added recordTags relevant to 1.0.1 in  mmd.json

### CODE REVIEW

**Comment by cs-rohit-khune** [comment]
@cs-suyog-jinturkar Please review this

File: modules/scenario/mmd.json [hunk 1]
```diff
@@ -17,7 +17,56 @@
...
+                "FSRSolutionPack",
+                "socSimulator-1.0.1"
+            ],
+            "displayName": "{{ steps }}",
+            "descriptions": {
+                "singular": "Steps"
+            }
+        },
+        {
+            "@id": "\/api\/3\/attribute_metadatas\/840d94ab-51b8-4782-ab37-5b0bdbfcbf69",
```
Comment (cs-suyog-jinturkar) [inline modules/scenario/mmd.json:69]: Can you check format of this id value?

File: info.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "publisher": "Fortinet",
     "description": "SOC Simulator Solution Pack",
     "summary": null,
-    "help": "https:\/\/github.com\/fortinet-fortisoar\/solution-pack-incident-response\/blob\/develop\/README.md",
+    "help": "https://github.com/fortinet-fortisoar/solution-pack-soc-simulator/blob/develop/README.md",
```
Comment (cs-suyog-jinturkar) [inline info.json:20]: I earlier one was correct. Have you changed this by hand?

Comment (cs-rohit-khune) [reply info.json:20]: Yes @cs-suyog-jinturkar , I have changed it by hand now, 
the earlier one was pointing to 'solution-pack-incident-response' readme.md, which is now corrected to solution-pack-soc-simulator readme.md

File: modules/scenario/mmd.json (hunk 1 above)
Comment (cs-rohit-khune) [reply modules/scenario/mmd.json:69]: @cs-suyog-jinturkar  Yes, this is the correct format.

**Pull request merged**